hera_mc/cm_hookup.py

```diff
@@ -270,54 +268,42 @@ def __get_part_types_found(self, huco, part_types_found):
 
     def __get_pols_to_do(self, part, port_query):
         """
-        Given the current part and port_query (which is either 'all', 'e', or 'n')
-        this figures out which pols to do.  Basically, given 'all' and part it
+        - 1 - Signal Path
+        Given the current part and port_query (which is either 'pol' (or 'all'), 'e', or 'n')
+        this figures out which pols to do.  Basically, given 'pol' and part it
         figures out whether to return ['e'], ['n'], ['e', 'n']
 
+        - 2 - Physical
+        If the port_query is '@' or '[phys]ical' it returns all physical ports.  Doesn't fully work.
```

Comment:
I do not like this magical `@` sigil at all because the non-expert cannot guess its meaning. Can we replace it with something short but meaningful?

Also, can the comment "doesn't fully work" be expanded to indicate what aspect doesn't work and why not?

Reply:
I got rid of this as a hookup option (i.e. lessened my ambitions - to see these connections use parts.py conn.  Maybe I'll re-add them to hookup, but likely not.)